scripts/manage_tx.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+
+import json
+import os
+import re
+from argparse import ArgumentParser
+from pathlib import Path
+
+from transifex.api import transifex_api
+
+# Version number of a new versioned project to create
+new_project_version = '3.11'
```

Comment:
nit: I wonder if it wouldn't be easier to accept this value from script argument

Reply:
Yeah, that was my initial desire when I created it (i.e. when bumping python-newest to 3.12), but I didn't want to expend more time on argparse at the time. Any feedback is welcome

Reply:
I think [Fire](https://google.github.io/python-fire/) (with version 1 or 4 from [hello world docs](https://google.github.io/python-fire/guide/#version-1-firefire)) should be feasible replacement for argparse that would handle this one extra argument for selected command. Should be more minimal than e.g. Typer or Click which have more verbose API.

Reply:
I liked Fire, but it looks like it is not compatible with 3.11 (and because a small fix with PR open and not merged yet). I've came across with Typer and Click before. I guess this could be a good time to play with them.